box-office.rb

```diff
@@ -1,5 +1,12 @@
 require 'sinatra'
+require 'httparty'
 
 get '/' do
   'Hello world!'
 end
+
+get '/movies/:imdbid' do |id|
+  response = HTTParty.get("http://www.omdbapi.com/?apikey=#{ENV['OMDB_API_KEY']}&i=#{id}&plot=full&r=json")
```

Comment:
let's keep it for this PR, we can change it later when we need. It doesn't hurt DRY not just yet (because even if you extract code, you would use the code only once)